File: backend-python/utils/logger.py
# backend-python/utils/logger.py
import time

class Logger:
    def log(self, msg):
        print(f"[LOG {time.strftime('%H:%M:%S')}] {msg}")

    def warn(self, msg):
        print(f"[WARN {time.strftime('%H:%M:%S')}] {msg}")

    def error(self, msg):
        print(f"[ERROR {time.strftime('%H:%M:%S')}] {msg}")

logger = Logger()

File: go-backend/core/logger.go
// core/logger.go
package core

import (
    "log"
    "time"
)

func Log(msg string) {
    log.Printf("[NeuroEdge] %s %s", time.Now().Format(time.RFC3339), msg)
}

func Error(msg string) {
    log.Printf("[NeuroEdge][ERROR] %s %s", time.Now().Format(time.RFC3339), msg)
}
